database-plugins/src/main/java/io/cdap/plugin/db/batch/source/DBSource.java

```diff
@@ -357,4 +358,16 @@ private Schema getSchema(FailureCollector collector) {
       throw collector.getOrThrowException();
     }
   }
+
+  private void emitLineage(BatchSourceContext context) {
+    Schema schema = sourceConfig.getSchema(context.getFailureCollector());
+    LineageRecorder lineageRecorder = new LineageRecorder(context, sourceConfig.referenceName);
+    lineageRecorder.createExternalDataset(schema);
+
+    if (schema != null && schema.getFields() != null) {
+      lineageRecorder.recordRead("Read", "Read from DB.",
+          schema.getFields().stream().map(Schema.Field::getName).collect(
```

Comment:
nit - the indentation is wrong. It should align with "Read" in the previous line